docs/developers/releasing.rst

```diff
@@ -148,7 +148,7 @@ quality checks.
 
 .. code-block:: bash
 
-    $ pootle regenerate_checks_descriptions
+    $ DJANGO_SETTINGS_MODULE=pootle.settings ./setup.py build_checks_templates
```

Comment:
I asked if it is possible to include the exporting of `DJANGO_SETTINGS_MODULE` within `setup.py` instead of prepending it to every call of this command. We have `os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'` in several management commands, including the one this PR replaces.